platform/MCU/shared/STM32/src/hw_ticks.c

```diff
@@ -39,8 +39,15 @@ static volatile system_tick_t system_millis_clock = 0;
  */
 void System1MsTick(void)
 {
+	int is = __get_PRIMASK();
```

Comment:
This is in the platform layer - platform doesn't depend on hal or wiring, so those options are not available.